flaskbb/management/forms.py

```diff
@@ -475,6 +487,14 @@ class ForumForm(FlaskForm):
 
     submit = SubmitField(_("Save"))
 
+    def validate_subforum_parent_id(self, field):
+        if not self.is_subforum:
+            self.subforum_parent_id = None
+        else:
+            if self.subforum_parent_id == None:
```

Comment:
```suggestion
            elif not self.subforum_parent_id:
```
В питоне есть конструкция elif, вместо ебучего нестед else: if
Ну если сабфорум_парент_айди не может быть 0 или нуллом - можно и if not юзать